eqcorrscan/core/lag_calc.py

```diff
@@ -396,7 +399,9 @@ def xcorr_pick_family(family, stream, shift_len=0.2, min_cc=0.4,
     return picked_dict
 
 
-def _prepare_data(family, detect_data, shift_len):
+def _prepare_data(family, detect_data, shift_len, all_vert=False,
+                  all_horiz=False, vertical_chans=['Z'],
+                  horizontal_chans=['E', 'N', '1', '2']):
```

Comment:
This probably won't come up as an issue because this is a private function, but it might be better if `horizontal_chans` is defined as the same default as in `xcorr_pick_family`.

Reply:
Actually, this is defined in a few places - if you don't have a reason for them not being consistent (it may be legacy from my inconsistency?)  then we should make them the same.